config/all-java.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<ruleset name="All Java Rules" 
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+    <description>Every Java Rule in PMD</description>
+
+    <rule ref="category/java/bestpractices.xml" />
+    <rule ref="category/java/codestyle.xml" />
+    <rule ref="category/java/design.xml" />
+    <rule ref="category/java/documentation.xml" />
+    <rule ref="category/java/errorprone.xml" />
+    <rule ref="category/java/multithreading.xml" />
+    <rule ref="category/java/performance.xml" />
+<!--     <rule ref="category/java/security.xml" /> -->
```

Comment:
we now have rules under security! :D https://pmd.github.io/pmd-6.3.0/pmd_rules_java_security.html